shell/app/common/constants.ts

```diff
@@ -51,3 +51,66 @@ export const DOC_CMP_CLUSTER_MANAGE = `${DOC_PREFIX}/manual/cmp/guide/cluster/ma
 // aliyun
 export const ALIYUN_APPLICATION =
   'https://account.aliyun.com/login/login.htm?oauth_callback=https%3A%2F%2Fdysms.console.aliyun.com%2Fdysms.htm%23%2Fdomestic%2Ftext%2Ftemplate%2Fadd';
+
+export const auxiliaryColorMap = {
```

Comment:
can these definition replace by the definition in theme-color?

Reply:
😂 Can't find a good solution for the time being

Reply:
😂 Can't find a good solution for the time being